v2.0/enable-node-map.md

```diff
@@ -4,4 +4,197 @@ summary: Learn how to enable the node map in the Admin UI.
 toc: false
 ---
 
-*Docs coming soon*
\ No newline at end of file
+The **Node Map** visualizes the geographical configuration of a multi-regional cluster by plotting the node localities on a world map. The **Node Map** also provides real-time cluster metrics, with the ability to drill down to individual nodes to monitor and troubleshoot the cluster health and performance.
+
+{{site.data.alerts.callout_info}}The <b>Node Map</b> is an <a href="enterprise-licensing.html">enterprise-only</a> feature. However, you can <a href="https://www.cockroachlabs.com/pricing/request-a-license/">request a trial license</a>  to enable the <b>Node Map</b>. {{site.data.alerts.end}}
+
+<div id="toc"></div>
+
+To enable the **Node Map**, you need to start the cluster with the correct `--locality` flags and assign the latitudes and longitudes for each locality.
```

Comment:
This content, right below the TOC and without its own section heading, is very likely to get lost/skipped. I'd suggest revising the intro to the page to incorporate this content:

```
The **Node Map** visualizes the geographical configuration of a multi-regional cluster by plotting the node localities on a world map. The **Node Map** also provides real-time cluster metrics, with the ability to drill down to individual nodes to monitor and troubleshoot the cluster health and performance.

This page walks your through the process of setting up and enabling the **Node Map**.

{{site.data.alerts.callout_info}}The <b>Node Map</b> is an <a href="enterprise-licensing.html">enterprise-only</a> feature. However, you can <a href="https://www.cockroachlabs.com/pricing/request-a-license/">request a trial license</a>  to try it out. {{site.data.alerts.end}}
```

Reply:
Reworked it a bit (not sure if it's enough).